### PR TITLE
Logo Update

### PR DESCRIPTION
**Pull request by Fish2**
#### Database Migration
NO

#### Description
Fixed the main SVG fully symmetrical now and also to match Sonarr, Radarr and Lidarr
Added my resource SVG for later use for any one wanting to add to this
Updated all the favicon's and modernised them
Added Read me Header
Removed some un-used image files
All images have been lossless compressed
Fixed all image links

#### Todos
- [ ] Tests
- [ ] Documentation

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

**Comment by Fish2** [comment]
@danielunderwood no must be my Mac adding that

File: frontend/src/Content/Images/logo.svg [hunk 1]
```diff
@@ -1 +1,20 @@
.../www.w3.org/2000/svg" width="1024" height="1024"><style>.a{fill:#989898;stroke-width:24;}.b{fill:none;stroke-width:16;stroke:#009252;}</style><defs><clipPath><circle cx="512" cy="512" r="505" style="fill:#989898;stroke-width:3.6"/></clipPath><clipPath><path d="m512 636l0-565 -330 565 330 0z" class="a"/><path d="m512 388l0 565 330-565 -330 0z" class="a"/></clipPath></defs><g><title>  background</title><rect x="-1" y="-1" width="1026" height="1026" fill="none"/></g><g><title>  Layer 1</title><circle cx="512" cy="512" r="410" stroke-width="1.8"/><ellipse cx="512" cy="512" rx="460" ry="460" style="fill:none;stroke-width:99;stroke:#e5e5e5"/><ellipse cx="512" cy="512" rx="270" ry="270" style="fill:#e5e5e5;stroke-width:76;stroke:#e5e5e5"/><ellipse ry="410" rx="410" cy="512" cx="512" style="fill-opacity:null;fill:none;stroke-width:12;stroke:#009252"/><path d="m512 636l0-565 -330 565 330 0z"/><path d="m512 388l0 565 330-565 -330 0z" fill="#000"/><path d="m512 636l0-565 -330 565 330 0z" class="b"/><path d="m512 388l0 565 330-565 -330 0z" class="b"/><ellipse cx="512" cy="512" rx="150" ry="150" style="fill-opacity:null;fill:#009252;stroke-opacity:null;stroke-width:null;stroke:null"/></g></svg>
```
Comment (danielunderwood) [inline frontend/src/Content/Images/logo.svg]: This file still has executable permissions.

File: Logo/svg-resources/Lidarr-Ghost.svg [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (danielunderwood) [inline Logo/svg-resources/Lidarr-Ghost.svg]: Executable permissions on this file.

File: Logo/svg-resources/Lidarr-Text.svg [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (danielunderwood) [inline Logo/svg-resources/Lidarr-Text.svg]: Executable permissions on this file.

File: Logo/svg-resources/Lidarr-White.svg [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (danielunderwood) [inline Logo/svg-resources/Lidarr-White.svg]: Executable permissions on this file.

**Comment by Fish2** [comment]
@danielunderwood all fixed sorry about that

File: Logo/svg-resources/Lidarr-Text.svg [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
...xEjCgIB/f+rrKwFNHmr+/AAAAAAAQBN/mgBiAWeABYAJkASGABEFRcVAhQICwYGBAQBAQAIAD8yETMRMy/NPz8BENb9xjAxISMGBhUUMzI3BwYjIiY1ND4CNyMRMwGITzoyRyVBBUdOQVIPKyAnArxGVy9cGVovWU0kO0EnKwWeAAAAAAIAOv5oAYQFlAANACQAMEAVJgkkUgIiJSQAGRQUEhIiDw8OBgUNuAE3AD/NPzMRMzMRMy/NPwEQ1jL9Ms4wMRImNTQ2MzIXFhUUBwYjEyMGBhUUMzI3BwYjIiY1ND4CNyMRM9xKSi8xJSMjJTFWOzoyRyVBBUdOQVIPKyAnBqwEpEYzMUYjITM1ISP7XEZXL1wZWi9ZTSQ7QScrBBAAAAACALAAAAGiBxAADQARACBADRMQRAsEDhIRCA4CAAi4ATYAP80/PwEQ1szM/cYwMQEyFxYVFAcGIyImNTQ2AzMRIwEpMSUjIyUxL0pKLry8BxAjITM1ISNGMzFG/o76YgABALUAAAFhBBAAAwAVQAkFAVIDBAIGAQAAPz8BENb9zjAxEzMRI7WsrAQQ+/AAAAACALT/IgO5BZ4AAwATACRAEhFEExUJDAJEABQSAgmXAggBAgA/Pz8/ARDW/cYyEN7hMDETMxEjARQOAgcmJic+AzURM7S8vAMFGjthRh0/HzZHKxO8BZ76YgFidaSAazwZOCA2UVx4XARUAAAEAIL+XwNfBaMAEwAnACsAOwA+QB89GTtSNCM5BSlSDys8OgA0NDGwKgYoACMeFKwPCgCsAD/NOT/NOT8/PzN8Lxg/ARDWMv0y1jLG/TLGMDETMh4CFRQOAiMiLgI1ND4CITIeAhUUDgIjIi4CNTQ+AgEzESMlFA4CByYmJz4DNREz9hgpHhISHykXGCogEhIgKgIQGCkeEhIfKRcYKiASEiAq/cisrAKlFzdZQBw6HTFDKBKsBaMSHioYFyogEhIfKhgYKh4SEh4qGBcqIBISHyoYGCoeEv5t+/BzbZh1YzcXMx0zTFVvVAOzAAIAuP2bBKUFtAAKABoAaUAoAgIBBAUEmwMCFAMCBAMcAgEACgIKAZsAChQACgEACkQHGgVEBxsAGbgBOUAMDrgJAgYICgUDCAADAD8/OTk/Pz/tAS8Q3v3MEO0QAMGHBSsQAMGHBX0QxAEYENYAwYcFK4cIfcQwM
```
Comment (Qstick) [inline Logo/svg-resources/Lidarr-Text.svg]: Not sure I’m a fan of the partial border (only around the text). Also, would be nice to simplify/modernize the logo. Maybe ditch the grid and play with the colors.

**Comment by Fish2** [comment]
@Qstick Ok that sound like a good plan I have started a new repo for the development of the logo including Sonarr and Radarr hopefully we can unify them all https://github.com/Fish2/Sonarr-Radarr-Lidarr-Logo-Concepts

**Comment by Qstick** [comment]
@Fish2 Radarr has gone a different direction with the logo <
http://blog.radarr.video/development/update/2018/02/08/1-year-of-radarr.html>

**Comment by Qstick** [comment]
Also that green is very bright on the Lidarr mockup. Lol

**Comment by Fish2** [comment]
@Qstick Nice that looks really smart

**Comment by Qstick** [comment]
Closing, stale

**Pull request closed without merging**